docs/contributors.rst

```diff
@@ -27,11 +27,8 @@ For more information on this, see the `quickstart guide
 Help with hacking!
 ==================
 
-First step is to set up Kitsune so you can run it and hack on it. For
-that, see the :ref:`hacking-howto-chapter`.
-
-If you have problems, please let us know! See
-:ref:`contact-us-chapter`.
+First step is to set up Kitsune so you can run it and hack on it.
+If you have problems, please let `us <https://github.com/mozilla/kitsune/graphs/contributors>`_ know!
```

Comment:
Cant understand why refer to this page if the person has problem? :disappointed: